rocky/rocky/bytes_client.py

```diff
@@ -88,22 +87,23 @@ def upload_raw(self, raw: bytes, manual_mime_types: set[str], input_ooi: str | N
         self._save_raw(boefje_meta.id, raw, {"boefje/manual"}.union(manual_mime_types))
 
     def _save_boefje_meta(self, boefje_meta: BoefjeMeta) -> None:
-        response = self.session.post(f"{self.base_url}/bytes/boefje_meta", data=boefje_meta.json())
+        response = self.session.post(
+            "/bytes/boefje_meta", content=boefje_meta.json(), headers={"content-type": "application/json"}
```

Comment:
The `json()` method returns a string, so we should use `data` instead of `content`. `json()` is also deprecated, so we can also change this to `model_dump_json` while we are changing this code.